README.rst

```diff
@@ -150,6 +151,40 @@ Android:
    compile ('org.web3j:core:3.3.1-android')
 
 
+Snapshot dependencies
+---------------------
+
+If you would like to use snapshots instead please add a new maven repository pointing to:
```

Comment:
Please can you specify what a snapshot version name looks like?

Reply:
Done - please let me know if you'd like further refinements/changes. I'll take a look at the web3j docs as well.